src/transformers/utils/quantization_config.py

```diff
@@ -388,11 +391,12 @@ def __init__(
         self.pad_token_id = pad_token_id
         self.disable_exllama = disable_exllama
         self.max_input_length = max_input_length
+        self.disable_exllamav2 = disable_exllamav2
         self.post_init()
```

Comment:
Can you add a check in `post_init` to make sure users can't set both `disable_exllama` and `disable_exllama_v2` (i.e. enable both) ?

Reply:
sounds good